pvt/helpers/src/models/types/TypesConverter.ts

```diff
@@ -58,47 +69,106 @@ export default {
+      weights,
+      rateProviders,
+      assetManagers,
+      swapFeePercentage,
+      pauseWindowDuration,
+      bufferPeriodDuration,
+      owner: params.owner,
+      from: params.from,
+    };
+  },
+
+  toLiquidityBootstrappingPoolDeployment(
+    params: RawLiquidityBootstrappingPoolDeployment
+  ): LiquidityBootstrappingPoolDeployment {
+    let { tokens, weights, swapFeePercentage, swapEnabledOnStart, pauseWindowDuration, bufferPeriodDuration } = params;
+    if (!params.owner) params.owner = ZERO_ADDRESS;
+    if (!tokens) tokens = new TokenList();
+    if (!weights) weights = Array(tokens.length).fill(fp(1));
+    weights = toNormalizedWeights(weights.map(bn));
+    if (!swapFeePercentage) swapFeePercentage = bn(1e16);
+    if (!pauseWindowDuration) pauseWindowDuration = DEFAULT_PAUSE_WINDOW_DURATION;
+    if (!bufferPeriodDuration) bufferPeriodDuration = DEFAULT_BUFFER_PERIOD_DURATION;
+    if (swapEnabledOnStart == undefined) swapEnabledOnStart = true;
+
+    return {
+      tokens,
+      weights,
+      swapFeePercentage,
+      swapEnabledOnStart,
+      pauseWindowDuration,
+      bufferPeriodDuration,
+      owner: params.owner,
+      from: params.from,
+    };
+  },
+
+  toManagedPoolDeployment(params: RawManagedPoolDeployment): ManagedPoolDeployment {
+    let {
+      tokens,
+      weights,
+      rateProviders,
+      assetManagers,
+      swapFeePercentage,
       swapEnabledOnStart,
       mustAllowlistLPs,
       managementAumFeePercentage,
-      poolType,
-      aumFeeId,
+      pauseWindowDuration,
+      bufferPeriodDuration,
       factoryVersion,
       poolVersion,
+      poolType,
+      aumFeeId,
     } = params;
     if (!params.owner) params.owner = ZERO_ADDRESS;
     if (!tokens) tokens = new TokenList();
     if (!weights) weights = Array(tokens.length).fill(fp(1));
     weights = toNormalizedWeights(weights.map(bn));
     if (!swapFeePercentage) swapFeePercentage = bn(1e16);
-    if (!pauseWindowDuration) pauseWindowDuration = 3 * MONTH;
-    if (!bufferPeriodDuration) bufferPeriodDuration = MONTH;
     if (!rateProviders) rateProviders = Array(tokens.length).fill(ZERO_ADDRESS);
     if (!assetManagers) assetManagers = Array(tokens.length).fill(ZERO_ADDRESS);
-    if (!poolType) poolType = WeightedPoolType.WEIGHTED_POOL;
-    if (undefined == aumFeeId) aumFeeId = ProtocolFee.AUM;
+    if (!poolType) poolType = ManagedPoolType.MANAGED_POOL;
+    if (swapEnabledOnStart == undefined) swapEnabledOnStart = true;
     if (undefined == swapEnabledOnStart) swapEnabledOnStart = true;
     if (undefined == mustAllowlistLPs) mustAllowlistLPs = false;
     if (undefined == managementAumFeePercentage) managementAumFeePercentage = FP_ZERO;
     if (undefined == factoryVersion) factoryVersion = 'default factory version';
     if (undefined == poolVersion) poolVersion = 'default pool version';
+    if (undefined == aumFeeId) aumFeeId = ProtocolFee.AUM;
+    if (!pauseWindowDuration) pauseWindowDuration = 9 * MONTH;
```

Comment:
Is 9 months the default for managed pools? I can't find that before the refactor.

Reply:
That's what I deployed it with. Yes; we wanted to make the pause window longer for Managed, since they're new, and without a controller, there won't be "production" pools for a while: we don't the pause window to expire before any pools are created.